individual/tests/graphql_mutation_group_test.py

```diff
@@ -175,24 +180,27 @@ def test_update_group_row_security(self):
 
         # SP officer B cannot update group for district A
         response = self.query(query_str)
+        content = json.loads(response.content)
+        internal_id = content['data']['updateGroup']['internalId']
+        self.assert_mutation_error(internal_id, 'mutation.authentication_required')
```

Comment:
This shouldn't be necessary as the anonymous user case is already tested in the test above. I see why you added them – probably due to the extraneous line I have above where the response was not checked. I'll push a change to remove this and similar changes below.

Reply:
Yes, I removed the query